src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.logo {
+  max-width: 40px;
+  max-height: 40px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-right: 10px;
+  align-items: center;
+  text-transform: uppercase;
+}
+
+.nav__link {
```

Comment:
The height of the .nav__link is hardcoded. It's better to use padding for spacing inside the link to make it more flexible.

Reply:
this is also ins't fixed

Reply:
I slightly don't understand what indentation value should be set to pass all the tests, and whether it is only needed in this tag.